.github/workflows/cmake-macos.yml

```
@@ -19,51 +20,82 @@ jobs:
-        os: ["macos-12"]
-        python-version: ["3.7.9"]
+        os: ["macos-14"]
+        python-version: ["3.11"]
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
       with:
         ref: ${{ github.ref }}
         submodules: 'recursive'
+    - name: Select current SDK
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:1:22: Quote this to prevent word splitting

(shellcheck)